pkg/buildkitutil/buildkitutil_unix.go

```diff
@@ -0,0 +1,50 @@
+//go:build freebsd || linux
```

Comment:
`//go:build unix` maybe?

Reply:
I just copied this from another pair of files that was split between `_windows.go` and `_unix.go`. Looking around, I see this was added in Go 1.19, but not added for filenames, which is mildly annoying, but I understand the rationale.

So yeah, that makes sense, as long as we're not expecting to support go 1.18. It's not yet used in the codebase, but I did see one `!windows` which I had never thought of... So probably worth fixing project-wide.